desc/compute/_neoclassical.py

```diff
@@ -272,3 +278,253 @@ def eps_32(data):
 def _effective_ripple(params, transforms, profiles, data, **kwargs):
     data["effective ripple"] = data["effective ripple 3/2"] ** (2 / 3)
     return data
+
+
+@register_compute_fun(
+    name="Gamma_c Velasco",
+    label=(
+        # Γ_c = π/(8√2) ∫dλ 〈 ∑ⱼ [v τ γ_c²]ⱼ 〉
+        "\\Gamma_c = \\frac{\\pi}{8 \\sqrt{2}} "
```

Comment:
is this correct? I thought this was for nemovs?

Reply:
They are the same except for the definition of γ_c